functions/index.js

```diff
@@ -61,6 +61,50 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
     });
 });
 
+exports.startNewSprint = functions.https.onRequest((request, response) => {
+    cors(request, response, () => {
+        console.log(request);
+
+        var status = request.body.data.Status;
+        var startDate = request.body.data.Startdate;
+        var endDate = request.body.data.Enddate;
```

Comment:
Change the variable name: Startdate and Enddate

Reply:
changed